model/src/order.rs

```diff
@@ -399,8 +402,9 @@ mod tests {
             "uid": "0x1111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111",
             "availableBalance": "100",
             "executedBuyAmount": "3",
-            "executedSellAmount": "4",
-            "executedFeeAmount": "5",
+            "executedNetSellAmount": "4",
+            "executedGrossSellAmount": "5",
+            "executedFeeAmount": "6",
```

Comment:
Are these numbers alright? if the fee is 6, then shoudln't be the gross net+fee: `4 + 6 = 10` <> `5`

Reply:
They are not correct. The issue is that the datatype doesn't enforce that the values are interdependent (and this unit test only tests the encoding), since I didn't find a good way to make serde (the serialisation framework) derive an extra value from a struct (other than using an inner struct which would then itself have the same issue).